diff --git a/node_modules/@nacelle/nacelle-nuxt-module/lib/build/data/getContentDataItems.js b/node_modules/@nacelle/nacelle-nuxt-module/lib/build/data/getContentDataItems.js
index c1ab36f..2bfd23e 100644
--- a/node_modules/@nacelle/nacelle-nuxt-module/lib/build/data/getContentDataItems.js
+++ b/node_modules/@nacelle/nacelle-nuxt-module/lib/build/data/getContentDataItems.js
@@ -4,18 +4,20 @@ export default async ({ client, routeConfig }) => {
   const content = await client.data.allContent()
 
   if (content && content.length > 0) {
-    const items = content.map(item => {
-      item.source = item.cmsSyncSource
-
-      return createDataItemObject({
-        type: item.type,
-        item,
-        routeConfig
-      })
-    })
 
+    const indexableContent = ['article', 'page', 'blog', 'collection'];
+    let items = []
+    for (const item of content) {
+      if (indexableContent.includes(item.type)) {
+        let ret = createDataItemObject({
+          type: item.type,
+          item,
+          routeConfig
+        })
+        items.push(ret)
+      }
+    }
     return items
   }
-
   return []
 }
